server/util/id/id.go

```diff
@@ -0,0 +1,17 @@
+package id
+
+import (
+	//MD5 is fine here, it's just making an ID and is not a security application
+	"crypto/md5" // #nosec G501
+	"encoding/binary"
+)
+
+func InvocationPKFromID(iid string) int64 {
```

Comment:
Since these only look used in one file, might want to avoid the extra indirection of this util file and add nosec directives there?

Reply:
They are used in two files: `server/build_event_protocol/build_event_handler/build_event_handler.go` and  `server/build_event_protocol/target_tracker/target_tracker.go`, so I decided to factor it out. The md5Int64 function they replace was copied from one file to the other at some point.

Reply:
Ah I see. I missed that InvocationPKFromID was used in 2 places.